settings.py

```diff
@@ -2,36 +2,30 @@
 # -*- coding: utf-8 -*-
 
 from wger.settings_global import *
-from decouple import config
-import dj_database_url
-import os
```

Comment:
Why remove these? @solnsubuga doesn't _heroku_ need `dj_database_url`?

Reply:
@lym and @Yiga-fred these required to run the application on heroku.